extensions-builtin/Lora/lora.py

```diff
@@ -248,9 +248,12 @@ def load_loras(names, multipliers=None):
         lora_on_disk = loras_on_disk[i]
 
         if lora_on_disk is not None:
+            if lora is None:
+                lora = cached_loras.get(name,None)    

```

Comment:
There's `loaded_loras` already..?

Reply:
Yes, but it only caches LoRa from last prompt. If this webUI serves different users at the same time it will cost much time loading LoRas from disk because they may use different LoRas. Maybe we can use LRU to prevent memory leak.